app/models/apiModels/user.js

```diff
@@ -125,7 +122,7 @@ export class UserUpdate extends BaseModelUpdate {
  */
 export class UserDelete extends BaseModel {
   static schema = {
-    username: { type: string, required: true },
+    email: { type: string },
```

Comment:
all delete model fields should be required
```suggestion
    email: { type: string, required: true },
```